assets/javascripts/legal-summaries.js

```diff
@@ -0,0 +1,29 @@
+(function($) {
+
+  var summarise = function(){
+    var $div = $(this);
+    var $firstP = $('p:first-of-type', $div);
+    var $extraElements = $firstP.nextAll();
+
+    if($extraElements.length > 0){
+
+      if($('html').attr('lang') == 'es'){
+        var readMore = '(Ver mas)';
+      } else {
+        var readMore = '(Read more)';
```

Comment:
My JSLint is unhappy about this definition inside an `if`. Could you switch this to:
```javascript
var readMore = '(Ver mas)';
if($('html').attr('lang') == 'en'){
       readMore = '(Read more)';
}
(The logic then matches the site's language preferences too)